src/AnnotationDialog.js

```diff
@@ -1,20 +1,19 @@
 import EventEmitter from 'events';
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
 import * as util from './util';
 import * as constants from './constants';
-import { ICON_DELETE } from './icons/icons';
+
+import Annotation from './components/Annotation';
 
 const POINT_ANNOTATION_ICON_HEIGHT = 31;
 const POINT_ANNOTATION_ICON_DOT_HEIGHT = 8;
 const CLASS_FLIPPED_DIALOG = 'ba-annotation-dialog-flipped';
 
-const CLASS_CANCEL_DELETE = 'cancel-delete-btn';
 const CLASS_COMMENT = 'annotation-comment';
```

Comment:
These ones weren't added to constants because they were only used in this file? I don't really have a problem adding them all into constants later (in addition to adding all the new React classes as well)

Reply:
Sorry... confused this w/ hasComments which is another terrible boolean. Yes I can remove hasAnnotations :)

Reply:
Element classes should live in their respective components rather than a centralized constants file, in my opinion. It keeps things encapsulated and modular.